### PR TITLE
Updates air-side economizer documentation

### PR DESCRIPTION
**Pull request by asparke2**
Pull request overview
---------------------

Updates the reference documentation to describe the change in default economizer type and the addition of two economizer faults. These changes were previously made by @JanghyunJK, but were not yet included in the documentation.

### Pull Request Author: Andrew Parker

Type of pull request (select all the apply):
 - [ ] Sampling
 - [ ] Core model measure
 - [ ] End-Use Savings Shapes measure
 - [ ] Reporting measure
 - [ ] Singularity image / Buildstock batch
 - [ ] Postprocessing
 - [X] Documentation

 - [ ] Added tests for new measures
 - [ ] Updated measure .xml(s)
 - [ ] Register values added to `comstock_column_definitions.csv`
 - [ ] Both `options_lookup.tsv` files updated
 - [ ] 10k+ test run
 - [X] Change documentation written
 - [ ] Measure documentation written
 - [X] Comstock documentation updated
 - [ ] Changes reflected in example `.yml` files
 - [ ] Changes reflected in `README.md` files
 - [ ] Added 'See ComStock License' language to first two lines of each code file
 - [ ] All new and existing tests pass the CI

### Review Checklist

This will not be exhaustively relevant to every PR.
 - [ ] Perform a code review on GitHub
 - [ ] All related changes have been implemented: data and method additions, changes, tests
 - [ ] If fixing a defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [X] Reviewed change documentation
 - [ ] Ensured code files contain License reference
 - [X] Results differences are reasonable
 - [ ] CI status: all tests pass

### CODE REVIEW

File: documentation/reference_doc/4_9_hvac.tex [hunk 1]
```diff
@@ -238,15 +238,73 @@ \subsection{Air-Side Energy Recovery}
...
+
+Figure~\ref{fig:economizer_prevalence} shows the comparison of economizer coverage with respect to building floor area between ComStock and estimation from Commercial Buildings Energy Consumption Survey \citep{eia2018cbecs}. Because of how data is structured in CBECS, the floor area shown in these figures represents the entire floor area of the building if any economizer is present in any of the HVAC systems in the building rather than actual floor area covered by HVAC systems with an economizer. Because CBECS data only shows total building area instead of total area covered by the economizers, this comparison helps give a rough estimate of economizers.
```
Comment (JanghyunJK) [inline documentation/reference_doc/4_9_hvac.tex:263]: I haven't tried opening/compiling this to actual document view, but I think we need to add "left" is comstock and "right" is CBECS (for `fig:economizer_prevalence`)? or I can add "comstock" and "cbecs" strings in the figure directly?

File: documentation/reference_doc/figures/econ_damper_fault.png
Comment (JanghyunJK) [inline documentation/reference_doc/figures/econ_damper_fault.png:1]: oh looks like I don't have rights to commit on this branch. but I think it'd be better to remove those citations numbers from the image. I actually made that change in the change documentation so it'd be nice to replace this figure image with the image in the change doc.

File: documentation/reference_doc/figures/econ_temp_fault.png
Comment (JanghyunJK) [inline documentation/reference_doc/figures/econ_temp_fault.png:1]: same for this for removing citation number from the image.

File: documentation/reference_doc/figures/econ_temp_fault_single_model.png
Comment (JanghyunJK) [inline documentation/reference_doc/figures/econ_temp_fault_single_model.png:1]: also made change in the change doc for the first image in this figure. it's replacing the word "decrease" to "difference".

File: documentation/reference_doc/4_9_hvac.tex [hunk 2]
```diff
@@ -238,15 +238,73 @@ \subsection{Air-Side Energy Recovery}
...
...ulted (among applicable economizers) is less than the defined prevalence due to implementation limitations. For example, 35\% of randomly selected buildings that include certain HVAC system types (categorized by the air system) are assigned the damper fully closed fault. However, certain portions of these air systems do not have economizers, thus, they cannot have an economizer fault. In other words, the current limitation is that the random selection of faulted economizers is not fully aligned with buildings that actually have economizers. In these cases, we are losing the opportunity for applying faults and decreasing the representation of fault prevalence in final building stock. Additional to sampling limitations, another limitation is that the economizer faults are currently not applied to packaged single zone systems due to incorrect applicability implementation. Newer versions of California's Title 24 energy code requires fault detection and diagnostics (FDD) for economizers which should prevent and mitigate faults. However, ComStock does not reflect the impact of FDD technology, possibly overestimating the impact of faults in buildings with newer HVAC systems in California.
```
Comment (JanghyunJK) [inline documentation/reference_doc/4_9_hvac.tex]: My goal for this cycle is to resolve this below. I think we can remove this sentence assuming I'll resolve this in this cycle. But I'll defer that decision to you.

> Additional to sampling limitations, another limitation is that the economizer faults are currently not applied to packaged single zone systems due to incorrect applicability implementation.

**Review by JanghyunJK — DISMISSED** [review]
approving with minor change requests!

File: documentation/reference_doc/figures/econ_damper_fault.png
Comment (asparke2) [reply documentation/reference_doc/figures/econ_damper_fault.png:1]: Updated figure from change doc

File: documentation/reference_doc/figures/econ_temp_fault_single_model.png
Comment (asparke2) [reply documentation/reference_doc/figures/econ_temp_fault_single_model.png:1]: Updated figure from change doc

File: documentation/reference_doc/figures/econ_temp_fault.png
Comment (asparke2) [reply documentation/reference_doc/figures/econ_temp_fault.png:1]: Updated figure from change doc

File: documentation/reference_doc/4_9_hvac.tex (hunk 1 above)
Comment (asparke2) [reply documentation/reference_doc/4_9_hvac.tex:263]: Added note to caption

File: documentation/reference_doc/4_9_hvac.tex (hunk 2 above)
Comment (asparke2) [reply documentation/reference_doc/4_9_hvac.tex]: I believe in you! removed this limitation.

**Pull request merged**